src/main/resources/k8s/def/windup.deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       serviceAccountName: windup-operator
       containers:
-        - image: quay.io/windupeng/windup-operator-native:latest
+        - image: quay.io/windupeng/windup-operator-native:0.0.2
```

Comment:
Do we want to deploy this particular version?

Reply:
Good catch.
This file is only intended for manual deployment of the operator. So I thought it was wrong to have latest and we should have 0.0.2 or stable. 
We are going to get rid of the "stable" tag, but for the moment I will modify it as "stable" as it's the one really deployed by operatorhub.
wdyt ?

Reply:
Great, thanks.